compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -13,6 +13,24 @@ include "mlir/IR/OpBase.td"
 include "mlir/IR/AttrTypeBase.td"
 include "mlir/IR/EnumAttr.td"
 
+/// TODO: This is unfortunate...
```

Comment:
It looks like the upstream version only defines the enum, not the EnumAttr nor the ArrayAttr. I'm leaving the TODO here because it would be great to deduplicate the attribute definitions, but I'm not well versed enough in how enums vs attributes are generated with TableGen to know how to do it. For now I changed it to at least use `utils::IteratorType` for the underlying value.